### DISCUSSION TITLE
Android 11 Support

### DISCUSSION THREAD

**Discussion by Z0ckerGmbH** [Shabinder/SpotiFlyer#76] [General]
Hello there,
I really enjoyed the Android App on my last Smartphone. But I have gotten a new one which comes with android 11. Idk if it matters but it ist the Xiaomi mi 10 with MIUI 12. I recently wanted to install SpotiFleyer again. But sadly it doesn't work for me. Every time I want to install the APK I get a parsing error.
What can I do?
Thanks in Advance

**Comment by Shabinder** [comment]
Hey there, I also own a smartphone with android 11 , and I am.using Spotiflyer without any issue, even before releasing i test it on my device which is android 11 as stated earlier.

**Comment by Shabinder** [comment]
Try installing latest apk and report back how it goes.

**Comment by Z0ckerGmbH** [comment]
I tried v 2.2 again and there is the same Issue. Maybe there is a problem with MIUI?

**Reply by Shabinder** [reply]
> I tried v 2.2 again and there is the same Issue. Maybe there is a problem with MIUI?

I tried it on MIUI on Redmi Note 8, it was working fine.
I don't know what could be the issue....
by the way the package manager throws the error right?
i think miui package manager has some extra checks like trusted package etc , which might be enabled in your device , try checking that if any such thing exists..... 

Edit: Also try redownloading the apk with different browser/download manager.

**Reply by Shabinder** [reply]
![image](https://user-images.githubusercontent.com/44765050/112335712-79ddab00-8ce2-11eb-8256-18ce685587fc.png)
See I have MIUI users using this app,so that must mean they were able to install the app.

**Reply by Z0ckerGmbH** [reply]
Thanks for your fast Tipps. 
I redownload the APK and installed it with a 3rd party APK Installer from the Playstore. It seems like the Xiaomi APK Installer doesn't like some Apps.
I'm happy that your great App is working again.

**Reply by Shabinder** [reply]
I am glad you got it working.

**Reply by felix7droid** [reply]
Is the apk compatible with one ui android 10 ? The apk installs fine but there is no song downloading.As I press on download it immediately fails.

**Reply by Shabinder** [reply]
@felix7droid check if you have granted storage permission , also see if you are usibg latest app ,and is still doesnt workntry uninstalling and installing again
